### DISCUSSION TITLE
Industries that export goods

### DISCUSSION THREAD

**Discussion by drewtato** [voxeltycoon/issues#488] [General]
It would be nice if cities exported some materials, such as trash and mail. Trash could be sent to a landfill, which would work like a reverse resource that accepts trash until it is full. Mail could be a second "people" resource.

Trash could be a gateway resource. At a certain size, cities no longer grow unless you export their trash. This would incentivize removing trash without having it generate money.

There could also be an option of recycling trash to create less trash and some useful materials.

**Comment by Sliderpro** [comment]
Food industry with something green-looking came first to mind 🤔  popularizing healthy lifestyle.. 😄
